config/title_music/EXAMPLE.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name" : "Starman",
+	"author" : "David Bowie",
+	"file" : "title_music/sounds/starman.ogg",
+	"map" : false,
+	"rare" : false
+}
```

Comment:
Are json files supposed to have an EOL newline?

Reply:
I... have no idea.

Reply:
Gonna assume that Byond doesn't give a shit

Reply:
BYOND's json parser is lazy enough to not care.